### DISCUSSION TITLE
Can you give an example of how to make a button?

### DISCUSSION THREAD

**Discussion by KhandamovA** [TurboWarp/extensions#795] [Q&A] [answered]
Can you give an example of how to make a button? I got a button, but nothing happens when I click it, I specified a function in opcode

**Comment by yuri-kiss** [comment]
buttons do not use opcodes

**Reply by yuri-kiss** [reply, marked as answer]
look at this
https://github.com/TurboWarp/extensions/blob/master/website/test/button.js

**Reply by yuri-kiss** [reply]
and you should probably just trim the title to `Can you give an example of how to make a button?`.

**Reply by KhandamovA** [reply]
Thanks
